diag_test.py

```
@@ -1052,7 +1052,7 @@ def render_contingency_table_html(df: pd.DataFrame, row_var_name: str, _col_var_
     group_count = 0
     col_groups = []
     
-    for l0, _ in zip(col_level_0, col_level_1, strict=True):
+    for l0, _ in zip(col_level_0, col_level_1):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)